app/indexers/identity_metadata_indexer.rb

```diff
@@ -9,10 +9,10 @@ def initialize(cocina:, **)
 
   # @return [Hash] the partial solr document for identityMetadata
   def to_solr
-    return { 'objectType_ssim' => [object_type] } if object_type == 'adminPolicy' || cocina_object.identification.nil?
+    return { 'objectType_ssim' => object_types } if object_types.include?('adminPolicy') || cocina_object.identification.nil?
 
     {
-      'objectType_ssim' => [object_type],
+      'objectType_ssim' => object_types,
```

Comment:
This field is used by Argo to determine what partials to display and I'd be a bit hesitant to add values to it that are not expected.